### PR TITLE
Create Header components

### PR DESCRIPTION
**Pull request by Elfaram**
Create a header component (burgerMenu and mobile version call in the component NavBar)

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,21 @@
+
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Blank line

File: frontend/src/components/Burger/index.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <button
+        type="button"
+        onClick={() => {
+          setToggleMenu(!toggleMenu);
+          setOpen(!open);
+        }}
+        className="btn"
+      >
+        <div />
+        <div />
```
Comment (LuigiMenez) [inline frontend/src/components/Burger/index.jsx:35]: Why???

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
No line blank and why empty div tags

File: frontend/src/components/Burger/index.jsx (hunk 2 above)
Comment (Elfaram) [reply frontend/src/components/Burger/index.jsx:35]: Div=MenuBurger

**Review by Elfaram — COMMENTED** [review]
blank ok, div menuburger ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by ghost — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,13 +1,16 @@
+import Navbar from "@components/Navbar";
 import { Routes, Route } from "react-router-dom";
 import Profile from "@pages/Profile";
-
 import "./App.css";
 
 function App() {
   return (
+      <div className="App">
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Warning indentation

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
attention to the indentation

**Review by Elfaram — COMMENTED** [review]
Changed App.jsx indentation

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE, good job ;-)

**Pull request merged**